spec/factories/tutorials.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  sequence :title { |n| "Test Tutorial Title #{n}" }
+
+  factory :tutorial do
+    title
+
+    transient do
+      tutorial_step_count 2
+    end
+
+    after(:build) do |tutorial, evaluator|
+      tutorial.tutorial_steps = build_list(:tutorial_step, evaluator.tutorial_step_count, tutorial: tutorial)
+    end
+  end
```

Comment:
[This `factory_girl` issue](https://github.com/thoughtbot/factory_girl/issues/1008) helped make sense of creating tutorial steps in a tutorial factory.